chance/p5js/chance-praegnanz/index.js

```diff
@@ -0,0 +1,40 @@
+var rows;
+var columns;
+
+var xStep;
+var yStep;
+
+var heightoff;
+var widthoff;
+
+var positions = [];
+
+function setup(){
+  createCanvas(500, 500);
```

Comment:
You need to assign the canvas to a specific element so the site can render it directly.  Like this:  


```js
  var canvas = createCanvas(500, 500); // !this is important
  canvas.parent('sketch'); // !this as well
```